NRediSearch/Schema.cs

```diff
@@ -15,7 +15,8 @@ public enum FieldType
         {
             FullText,
             Geo,
-            Numeric
+            Numeric,
+            TAG
```

Comment:
is tag an abbreviation? or is this literally a "tag"? If the latter: `Tag` (capitalization) would be preferred